mirascope/chat/types.py

```diff
@@ -82,6 +84,14 @@ def tool(self) -> Optional[OpenAITool]:
 
         return None
 
+    def dump(self) -> dict:
```

Comment:
noticing you only have this for chat completion but not the chunks. Is it worth adding for the chunks?

Reply:
i wasnt sure yet, so i opted to not add it. it doesnt make sense to add chunks to db or log since its unreadable

Reply:
but we might want to add something such that when the chunks are all gathered, then make it easy to dump